pkgs/by-name/ri/rime-japanese/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "rime-japanese";
+  version = "0-unstable-2023-08-02";
+
+  src = fetchFromGitHub {
+    owner = "gkovacs";
+    repo = "rime-japanese";
+    rev = "4c1e65135459175136f380e90ba52acb40fdfb2d";
+    hash = "sha256-/mIIyCu8V95ArKo/vIS3qAiD8InUmk8fAF/wejxRxGw=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/rime-data/
+    cp japanese.*.yaml $out/share/rime-data/
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Layout for typing in Japanese with RIME";
+    homepage = "https://github.com/gkovacs/rime-japanese";
+    license = licenses.free; # awaiting response from gkovacs/rime-japanese#6, presumably free
```

Projects without a license should be presumed unfree: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

```suggestion
    license = licenses.unfree; # awaiting response from gkovacs/rime-japanese#6
```